website/core/Footer.js

```diff
@@ -79,33 +79,6 @@ class Footer extends React.Component {
             </a>
           </div>
         </section>
-
```

Comment:
@wing328 this removes attribution required for using these free tools. Is there a plan to attribute elsewhere?